server/manifestmergetool/src/main/java/com/defold/manifestmergetool/InfoPlistMerger.java

```diff
@@ -61,8 +72,14 @@ public static void mergePlists(XMLPropertyListConfiguration base, XMLPropertyLis
                             differ = true;
                         }
                         if (differ) {
-                            InfoPlistMerger.logger.log(Level.WARNING, String.format("Plist overriding value for key '%s': from '%s' to '%s'", key, baseValue.toString(), libValue.toString()));
-                            base.addProperty(key, libValue);
+                            String mergeMarker = mergeMarkers.get(key);
+                            if (mergeMarker == null || !mergeMarker.equals("keep")) {
+                                InfoPlistMerger.logger.log(Level.WARNING, String.format("Plist overriding value for key '%s': from '%s' to '%s'", key, baseValue.toString(), libValue.toString()));
+                                base.addProperty(key, libValue);
+                            }
+                            else {
+                                InfoPlistMerger.logger.log(Level.INFO, String.format("Plist keeping base '%s' value for key '%s'", baseValue.toString(), key));
+                            }
```

Comment:
I guess there still is no official way of doing this?

Reply:
I have not found one...